Recipes/Protocols.md

```diff
@@ -53,3 +53,80 @@ class MyClass: MyProtocol {
     }
 }
 ```
+
+### Solutions #3: Create a wrapper around the protocol
+If the protocol you want to confirm to is not under your control, and the protocol is non-isolated, you can create a non-concurrent wrapper around the protocol and then create an async protocol for your actor. For example, URLSession delegates that are not async.
+E.g:
+
+```swift
+// This could also be an actor instead of a class.
+@MainActor
+class MyClass {
+    var myClassProperty = "pew"
+    let someObject: Object
+
+    init() {
+        someObject = Object()
+        someObject.delegate = self // <----- We want to use the object delegate calls
+    }
+}
+
+extension MyClass: ObjectNonConcurrentDelegate { // <------- Not the best way ❗
+    // ERROR!: You would need to mark the method nonisolated like Solution #1.
+    // But sometimes we want to access our class properites, which means we would
+    // be accessing our actor from outside the actor system/boundries and this could lead to
+    // data races and threading issues, which is what we want to avoid
+    // when we use actors in the first place!
+    nonisolated func someMethod() {
+        myClassProperty = "blob"
+    }
+}
+
+// Wrapper Solution 👇🏼
+protocol MyWrapperConcurrentDelegate {
+    // Keep in mind this will only only if the paramters and/or
```

Comment:
"only only" -> "only"
"paramters" -> "parameters"

"and/or" actually must be "and" right?

Reply:
on it